analyzers/src/SonarAnalyzer.CSharp/Rules/AspNet/AvoidUnderPosting.cs

```diff
@@ -33,6 +33,9 @@ public sealed class AvoidUnderPosting : SonarDiagnosticAnalyzer
         KnownType.Microsoft_AspNetCore_Http_IFormCollection,
         KnownType.Microsoft_AspNetCore_Http_IFormFile,
         KnownType.Microsoft_AspNetCore_Http_IFormFileCollection);
+    private static readonly ImmutableArray<KnownType> IgnoredAttributes = ImmutableArray.Create(
+        KnownType.System_Text_Json_Serialization_JsonRequiredAttribute,
+        KnownType.System_ComponentModel_DataAnnotations_RangeAttribute);
```

Comment:
What about https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonRequiredAttribute.htm

Reply:
What about [JsonIgnoreAttribute ](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonIgnoreAttribute.htm)

Reply:
and System.Text.Json as well: https://learn.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonignoreattribute?view=net-8.0

Reply:
I've added them all!